.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: release
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.13
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v1
+    
+    - name: Calculate version
+      uses: fnkr/github-action-git-bash@v1
+      with:
+        args: bash -c "echo ::set-env name=GANTRY_VERSION::$(git describe --always --long --dirty)"
+
+    - name: Run GoReleaser
+      uses: goreleaser/goreleaser-action@v1
```

Comment:
Is it standard to release all versions? I'd usually only make a release when there is a `vX.Y.Z` tag and otherwise only test build, what do you think?

Reply:
The first few lines should enforce that only `v...` tags are released:

https://github.com/ad-freiburg/gantry/blob/b192c47b2f9ef52fb40b07b5c3bb9a4167bf56bf/.github/workflows/release.yml#L1-L5